creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -531,6 +531,7 @@ class SnapbackSM {
         numSyncRequestsIssued = resp.numSyncRequestsIssued
         syncRequestErrors = resp.syncRequestErrors
 
+        // Error if > 50% syncRequests fail
```

Comment:
lets also add a more descriptive error here in the throw?

Reply:
also this if condition checks if there's more errors than num syncs issued right? shouldn't this be syncs issued / 2?

Reply:
fixed error msg

Reply:
also no bc `syncRequestsIssued` is the number of sync requests that are successfully issued, not attempted. so errors > successes here

Reply:
fixed